### PR TITLE
[LOOP-4681] initial meals main screen UI; tab bar updates

### PR DESCRIPTION
**Pull request by SwiftlyNoah**
https://tidepool.atlassian.net/jira/software/c/projects/LOOP/issues

[Video Demo Link](https://drive.google.com/file/d/1nj8babQs426NxxUV389Z5gpx3fay6BBr/view?usp=sharing)

### CODE REVIEW

**Review by ArwainK — APPROVED** [review]
LGTM! But I would still get approval for Swift best practices from Cameron or Pete.

File: LoopKit/Meal.swift [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    
+    public var absorptionTimeString: String {
+        let absorptionFormatter: DateComponentsFormatter = {
+            let formatter = DateComponentsFormatter()
+            formatter.collapsesLargestUnit = true
+            formatter.unitsStyle = .full
+            formatter.allowsFractionalUnits = true
+            formatter.allowedUnits = [.hour, .minute]
+            return formatter
+        }()
```
Comment (Camji55) [inline LoopKit/Meal.swift]: I might suggest making this `absorptionFormatter` a property of `Meal` directly. This is a computed property so this date formatter will get re-created every time it's called which is kind-of a heavy operation.

**Review by Camji55 — APPROVED** [review]
1 suggestion -- otherwise LGTM

File: LoopKit/Meal.swift (hunk 1 above)
Comment (SwiftlyNoah) [reply LoopKit/Meal.swift]: Good catch! Thanks Cameron!

**Pull request merged**